types/node/url.d.ts

```diff
@@ -65,6 +65,11 @@ declare module 'url' {
      * * Widely relied upon within the npm ecosystem.
      *
      * Use with caution.
+     *
+     * Deprecation of this API has been shelved for now primarily due to the the
```

Comment:
You have two "the"s at the end of this line.

Reply:
I did not write this, out of scope.

Reply:
fyi: came from master changes